demisto_sdk/commands/download/downloader.py

```diff
@@ -1435,7 +1462,10 @@ def download_existing_content_items(
         source_to_destination_mapping: dict[
             Path, Path
         ] = {}  # A mapping of temp file paths to target final paths
-        is_unified = content_item_entity_directory in (INTEGRATIONS_DIR, SCRIPTS_DIR)
+        is_unified = content_item_entity_directory in (INTEGRATIONS_DIR, SCRIPTS_DIR, LISTS_DIR)
+
+        if content_item_type == FileType.LISTS:
+            return self.download_new_content_items(content_object, output_path)
```

Comment:
I think it's better and more readable to handle this on the upper function - `write_files_into_output_path` - instead of here, by changing:
```python
content_item_name: str = content_object["name"]
content_item_entity: str = content_object["entity"]
```
```python
if content_item_exists:
    downloaded_files = self.download_existing_content_items(
        content_object=content_object,
        existing_pack_structure=existing_pack_structure,
        output_path=output_path,
        temp_dir=temp_dir,
    )

else:
    downloaded_files = self.download_new_content_items(
        content_object=content_object,
        output_path=output_path,
    )
```

To:
```python
content_item_name: str = content_object["name"]
content_item_entity: str = content_object["entity"]
content_item_type: FileType = content_object["type"]
```
```python
# We skip 'download_existing_content_items' logic for lists since 'smart-merge' is irrelevant for lists
if content_item_exists and content_item_type != FileType.LISTS:
    downloaded_files = self.download_existing_content_items(
        content_object=content_object,
        existing_pack_structure=existing_pack_structure,
        output_path=output_path,
        temp_dir=temp_dir,
    )

else:
    downloaded_files = self.download_new_content_items(
        content_object=content_object,
        output_path=output_path,
    )
```

(The note there explaining why we ignore it is important for future reference)